src/createServer.js

```diff
@@ -1,11 +1,242 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  let users = [];
+  let expenses = [];
+
+  app.use(cors());
+
+  app.get('/users', (req, res) => {
+    if (!users) {
+      res.statusCode = 200;
+
+      res.send([]);
+
+      return;
+    }
+
+    res.statusCode = 200;
+    res.send(users);
+  });
+
+  app.get('/users/:userId', (req, res) => {
+    const { userId } = req.params;
+    const foundUser = users.find(user => user.id === +userId);
+
+    if (!foundUser) {
+      res.sendStatus(404);
+
+      return;
+    }
+
+    res.send(foundUser);
+  });
+
+  app.post('/users', express.json(), (req, res) => {
+    const { name } = req.body;
+
+    if (!name) {
+      res.sendStatus(400);
+
+      return;
+    }
+
+    const newUser = {
+      id: Math.max(0, ...users.map(({ id }) => id + 1)),
```

Comment:
You can just create variable with number 1 and then just increase the value by 1
```
let idCounter = 1;
const newUser = {
id : idCounter++,
}
```